pep-NNN.rst

```diff
@@ -142,14 +142,17 @@ when the user access ``__version__`` and it is not otherwise defined.
 
 None
 
-Footnotes
-=========
+References
```

Comment:
"Footnotes" is the language in the PEP template, 50/50 on changing it.

Going to merge, but we might have to change back.

Reply:
Ah sorry I hadn't checked the template. I'll do that before doing anything else.